orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/HttpClientUtils.java

```diff
@@ -89,6 +92,13 @@ public long getRetryInterval() {
       return executionCount <= MAX_RETRIES;
     });
 
+    String proxyHostname = System.getProperty(JVM_HTTP_PROXY_HOST);
+    if (proxyHostname != null) {
+      int proxyPort = System.getProperty(JVM_HTTP_PROXY_PORT) != null ? Integer.parseInt(System.getProperty(JVM_HTTP_PROXY_PORT)) : 8080;
```

Comment:
Are you sure you shouldn't use `System.getenv()`? The above code will only find jvm properties, not system environment variables...

Reply:
Yeah, it's only jvm properties I'm after!